### PR TITLE
Made sdk/io magic compliant

### PR DESCRIPTION
**Pull request by marcusnaslund**
Changes no functionality. This is just to make magic happy.

### CODE REVIEW

File: source/sdk/io/native/FileWin32.ooc [hunk 1]
```diff
@@ -101,9 +98,7 @@ version(windows) {
 			// our definition of a file: neither a directory or a link
 			// (and no, FILE_ATTRIBUTE_NORMAL isn't true when we need it..)
 			(ffd, ok) := _getFindData()
-			return (ok) &&
-					(((ffd attr) & FILE_ATTRIBUTE_DIRECTORY) == 0) &&
-					(((ffd attr) & FILE_ATTRIBUTE_REPARSE_POINT) == 0)
```
Comment (davidhesselbom) [inline source/sdk/io/native/FileWin32.ooc]: Magic complained about this?

Comment (marcusnaslund) [inline source/sdk/io/native/FileWin32.ooc]: "Incorrect indentation", yes.

Comment (davidhesselbom) [inline source/sdk/io/native/FileWin32.ooc]: Okay... I think it read a lot more clearly before you put it all on one line, though. What if you keep it the way it was, but fix the indentation?

Comment (marcusnaslund) [inline source/sdk/io/native/FileWin32.ooc]: Yep. Done.

**Pull request merged**